pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -135,6 +135,7 @@ enum ProtocolVersion {
 	v9 = 9;  // Added end of topic notification
 	v10 = 10;// Added proxy to broker
 	v11 = 11;// C++ consumers before this version are not correctly handling the checksum field
+			 //Added get topic's last messageId from broker
```

Comment:
@zhaijack There was a merge issue here, `v11` was already taken in master and this PR should be adding `v12` now.